dev-libs/efl/efl-1.21.0.ebuild

```diff
@@ -0,0 +1,290 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit gnome2-utils pax-utils xdg-utils
+
+DESCRIPTION="Enlightenment Foundation Libraries all-in-one package"
+HOMEPAGE="https://www.enlightenment.org/"
+SRC_URI="https://download.enlightenment.org/rel/libs/${PN}/${P}.tar.xz"
+
+LICENSE="BSD-2 GPL-2 LGPL-2.1 ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="avahi +bmp dds connman debug doc drm +eet egl examples fbcon +fontconfig fribidi gif gles glib gnutls gstreamer harfbuzz hyphen +ico ibus jpeg2k libressl libuv luajit neon nls opengl ssl pdf physics postscript +ppm +psd pulseaudio raw scim sdl sound static-libs svg +system-lz4 systemd tga tiff tslib unwind v4l valgrind vlc vnc wayland webp X xcf xim xine xpresent xpm"
+
+REQUIRED_USE="
+	?? ( opengl egl )
+	?? ( opengl gles )
+	fbcon? ( !tslib )
+	gles? (
+		|| ( X wayland )
+		!sdl
+		egl
+	)
+	ibus? ( glib )
+	opengl? ( X )
+	pulseaudio? ( sound )
+	sdl? ( opengl )
+	vnc? ( X fbcon )
+	wayland? ( egl gles !opengl )
+	xim? ( X )
+"
+
+COMMON_DEPEND="
+	net-misc/curl
+	media-libs/libpng:0=
+	sys-apps/dbus
+	sys-apps/util-linux
+	sys-libs/zlib:=
+	virtual/jpeg:0=
+	avahi? ( net-dns/avahi )
+	connman? ( net-misc/connman )
+	drm? (
+		dev-libs/libinput
+		media-libs/mesa[gbm]
+		x11-libs/libdrm
+		x11-libs/libxkbcommon
+	)
+	egl? ( media-libs/mesa[egl] )
+	fontconfig? ( media-libs/fontconfig )
+	fribidi? ( dev-libs/fribidi )
+	gif? ( media-libs/giflib:= )
+	glib? ( dev-libs/glib:2 )
+	gles? ( media-libs/mesa[gles2] )
+	gstreamer? (
+		media-libs/gstreamer:1.0
+		media-libs/gst-plugins-base:1.0
+	)
+	harfbuzz? ( media-libs/harfbuzz )
+	hyphen? ( dev-libs/hyphen )
+	ibus? ( app-i18n/ibus )
+	jpeg2k? ( media-libs/openjpeg:0= )
+	libuv? ( dev-libs/libuv )
+	luajit? ( dev-lang/luajit:= )
+	!luajit? ( dev-lang/lua:* )
+	nls? ( sys-devel/gettext )
+	pdf? ( app-text/poppler:=[cxx] )
+	physics? ( sci-physics/bullet:= )
+	postscript? ( app-text/libspectre )
+	pulseaudio? ( media-sound/pulseaudio )
+	raw? ( media-libs/libraw:= )
+	scim? ( app-i18n/scim )
+	sdl? (
+		media-libs/libsdl2
+		virtual/opengl
+	)
+	sound? ( media-libs/libsndfile )
+	ssl? (
+		gnutls? ( net-libs/gnutls:= )
+		!gnutls? (
+			!libressl? ( dev-libs/openssl:= )
+			libressl? ( dev-libs/libressl:= )
+		)
+	)
+	svg? (
+		gnome-base/librsvg
+		x11-libs/cairo
+	)
+	system-lz4? ( app-arch/lz4 )
+	systemd? ( sys-apps/systemd )
+	tiff? ( media-libs/tiff:0= )
+	tslib? ( x11-libs/tslib:= )
+	unwind? ( sys-libs/libunwind )
+	valgrind? ( dev-util/valgrind )
+	vlc? ( >=media-video/vlc-3.0.0 )
```

Comment:
This is a minor nitpick. Please change this line to `media-video/vlc`. There's no other version available other than a version greater than `3.0.0`

Reply:
Right, 
https://packages.gentoo.org/packages/media-video/vlc still lists "2.2.8-r1 : 0" but the git log below indeed says it's dropped. I'll fix this, thanks!